Add plugin: Pasterly

easternkite opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/easternkite/Pasterly

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

Zachatoo commented:
Remove the `styles.css` file from your release and repo, since it's empty it's just unnecessary bloat that Obsidian would download.

[new Notice('Please set your Firebase Storage Bucket URL in settings first');](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L74), [new Notice('Firebase Storage is not initialized. Please check your settings');](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L104), [.setName('Firebase Storage Bucket URL')](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L190), [.setDesc('URL of your Firebase Storage bucket (e.g., gs://your-bucket.appspot.com)')](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L191), [containerEl.createEl('h2', {text: 'Pasterly Settings'});](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L187)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI), Firebase, URL, and the name of your plugin can be uppercase

[containerEl.createEl('h2', {text: 'Pasterly Settings'});](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L187)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)
[Only use headings under settings if you have more than one section.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

[containerEl.createEl('h2', {text: 'Pasterly Settings'});](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L187)
For section headings in settings use:
```ts
new Setting(containerEl).setName('name here').setHeading();
```

[this.initializeTimeout = window.setTimeout(() => {](https://github.com/easternkite/Pasterly/blob/666ecee896e6857c95a7a9da1b7f040f85d4ea2c/src/main.ts#L88-L91)
Events need to be registered, so they can be properly unloaded when the plugin is disabled. <https://docs.obsidian.md/Plugins/Events>

easternkite commented:
@Zachatoo 
I've completed the requested changes: removed the empty styles.css file, updated the UI strings to use sentence case, removed the top-level heading from the settings tab, and ensured events are properly registered.
https://github.com/easternkite/Pasterly/releases/tag/1.0.2

Let me know if there's anything else you'd like changed!

Zachatoo commented:
Looks like the bot doesn't think you've made all the requested changes, can you double check them? If you have done all the requested changes I can see what's going on.

easternkite commented:
@Zachatoo 
I've resolved the issue on my end by updating the release script to no longer include styles.css.
Please take a look at version 1.0.3 when you get a chance. Thanks!

Zachatoo commented:
The bot hasn't moved this forward because you've missed some places where you need to use sentence case.
